lib/helpers/utils.js

```diff
@@ -448,6 +458,7 @@ export const PROJECT_TYPE_ALIASES = {
   ],
   binary: ["binary", "blint"],
   oci: ["docker", "oci", "container", "podman"],
+  cocoa: ["cocoa", "cocoapods", "objective-c", "swift", "ios"],
```

Comment:
We already have a swift type with an alias for ios. Let's remove swift and ios from here.

Reply:
True, but will it run the new scanner when I use `-t ios`?
Or did I misunderstand the meaning of the aliases?

Reply:
Interesting. Does your PR run both the scanners for `-t ios`?

Reply:
But I just noticed it is a bit of a problem to have duplicate entries, because using `-t ios` doesn't generate my SBOM...

Reply:
> Interesting. Does your PR run both the scanners for `-t ios`?

Missed your reply when I posted the above -- no, it doesn't, I guess I only tested using `-t cocoapods`.